src/ChartTemplates/PieChart/PieChartTemplate.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { ChartPie } from '@patternfly/react-charts';
+
+export const PieChart = ({ ...props }) => {
+    return (
+        <ChartPie
+            ariaDesc={ props.ariaDesc }
+            ariaTitle={ props.ariaTitle }
+            constrainToVisibleArea={ props.constrainToVisibleArea }
+            data={ [{ x: 'Red Hat Enterprise Linux 8', y: 20 }, { x: 'Red Hat Enterprise Linux 7', y: 20 }, { x: 'Other', y: 60 }] }
```

Comment:
When possible abstract out data and legend stuff, makes the template a lil more usable, more plug n play, also legend data can be derived from this, no need to repeat the same info